### ISSUE TITLE
NTR: Dopamine low in blood or urine (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8314]
New term request Dopamine low in blood or urine (elthat_130220204417):

**Comment by pnrobinson** [comment]
@MickeySegal
I am not an expert, but see that the normal range for Dopamine is given as  0 to 30 pg/mL (195.8 pmol/L) by one [source](https://medlineplus.gov/ency/article/003561.htm). If this is true, then logically there cannot be a "reduced" value.
Please provide the clinical context?

**Comment by MickeySegal** [comment]
We have:
- https://www.ncbi.nlm.nih.gov/books/NBK1474/ "Pure autonomic failure / autonomic neuropathy... Plasma and urinary norepinephrine concentrations are greatly reduced, sometimes to 10% of normal; plasma dopamine concentrations are normal or low, rather than elevated as in DBH deficiency."
- https://www.omim.org/entry/618049 "urinary analysis showed increased levels of neurotransmitter metabolites, including homovanillic acid (HVA) and 5-hydroxyindoleacetic acid (5-HIAA), as well as decreased levels of norepinephrine and dopamine."

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000751
- label: Decreased urinary dopamine level
- parents: [Decreased urinary catecholamine concentration (HP:0045012)](https://hpo.jax.org/app/browse/term/HP:0045012)
- pmid: 
- def: The concentration of dopamine in the urine, normalized for urine concentration, is below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8314
- synonyms:
